include/osquery/sql.h

```diff
@@ -100,11 +116,26 @@ class SQL {
    */
   SQL(){};
 
- private:
+  // The key used to store hostname for annotateHostInfo
+  static const std::string kHostColumnName;
```

Comment:
My understanding is that `#define` is discouraged for constant definition due to no support for namespaces. Happy to do whatever is standard.